epub32/spec/epub-spec.html

```diff
@@ -523,7 +523,7 @@ <h2>EPUB Reading Systems</h2>
 						<p id="confreq-rs-xml-ns"> It MUST be a <a
 								href="https://www.w3.org/TR/2009/REC-XML-NAMES-20091208/#ProcessorConformance"
 								>conformant processor</a> as defined in [[!XML-NAMES]].</p>
-						<p id="confreq-rs-xml-base">It MUST be a conformant application as defined by [[!XMLBase]].</p>
+						<p id="confreq-rs-xml-base">It MUST be a conformant application as defined by [[!XMLBase]]. <span class="note">Note: browsers have removed support for [[!XMLBase]]. Authors are advised to avoid using this feature.</span></p>
```

Comment:
There's just something weird about introducing the word browsers here when we normally use reading system. I know browser core -> reading system, but it's not a connection made in the specs. What about maybe making that clearer, like: "Note: Authors are advised to avoid using this feature as the browser cores Reading Systems are built on no longer support [[XMLBase]]." ?

Reply:
I was wondering about that. I'll adjust the wording. 

What I should really do is try to make a test file. I'm doubting we have any support now.

Reply:
Changed to 

> [[HTML]] and [[SVG]] are removing support for [[!XMLBase]]

Which is more accurate, and expresses the risk authors face much better.